spec/controllers/categories_controller_spec.rb

```diff
@@ -1,5 +1,27 @@
 require 'spec_helper'
 
+# describe CategoriesController, "/create" do
```

Comment:
If you don't want these commented out tests anymore, then they should be removed from your commits